### PR TITLE
Prepare 1.9.2-dev.0001

### PR DESCRIPTION
**Pull request by JaniruTEC**
This PR prepares the internal release 1.9.2-dev.0001.
Please note that all commits up to (and including) 39650285cfee2771a2f65ac8d9a7a10357a70e68 are already on main. This PR serves as a way to discuss all recent changes.
Once it's merged, I will rebase the remaining changes onto main.

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -12,25 +12,34 @@
...
-	}
+    dependencies {
+        classpath dep.androidPlugin
+    }
 }
 
+apply from: rootProject.file('publishing.gradle')
+
 ext {
-    description = "The official Android SDK for integration with pCloud's API."
```
Comment (SailReal) [inline android/build.gradle:32]: Should we add here our own description?

**Review by SailReal — COMMENTED** [review]
LGTM :+1:

File: android/build.gradle (hunk 1 above)
Comment (JaniruTEC) [reply android/build.gradle:32]: That's already handled:
https://github.com/cryptomator/pcloud-sdk-java/blob/a8a8a3587b70bde5c7a14f13b588711fa3ddd66d/maven-publish.gradle#L10

**Comment by JaniruTEC** [comment]
[`latest-release`](https://github.com/cryptomator/pcloud-sdk-java/tree/latest-release) was at c6681ed3762fdb3bda63fc9c147a369d3812f7d9.

**Pull request merged**